libs/ledger-live-common/scripts/sync-families-dispatch.mjs

```diff
@@ -105,14 +115,37 @@ function genCoinFrameworkTarget(targetFile) {
 
     if (targetFile === "bridge/js.ts") {
       const bridgeFn = family + "CreateBridges";
+      const signer = family + "Signer";
+      const capitalizeFamily = family[0].toUpperCase() + family.slice(1);
       imports += `import { createBridges as ${bridgeFn} } from "${targetImportPath}";\n`;
-      exprts += `\n  ${family}: ${bridgeFn}(withDevice, network, makeLRUCache),`;
+      imports += `import { SignerFactory as ${capitalizeFamily}SignerFactory } from "@ledgerhq/coin-polkadot/lib/signer";\n`;
+      imports += `import * as ${signer} from "@ledgerhq/hw-app-${family}";\n`;
+      imports += `const signerFactory: ${capitalizeFamily}SignerFactory = async (deviceId: string) => {\n`;
+      imports += `  return await withDevicePromise(deviceId, (transport) =>\n`;
+      imports += `    of(new ${signer}.default(transport))\n`;
+      imports += `  );\n`;
+      imports += `};\n`;
+      imports += `const ${family} = ${bridgeFn}(signerFactory, network, makeLRUCache);\n`;
+      exprts += `\n  ${family},`;
     }
     if (targetFile === "cli-transaction.ts") {
       const cliToolsFn = family + "CreateCliTools";
       imports += `import ${cliToolsFn} from "${targetImportPath}";\n`;
       exprts += `\n  ${family}: ${cliToolsFn}(network, makeLRUCache),`;
     }
+    if (targetFile === "hw-getAddress.ts") {
+      imports += `import * as ${family}Signer from "@ledgerhq/hw-app-${family}";\n`;
+      imports += `import ${family}Resolver from "${targetImportPath}";\n`;
+      imports += `const ${family}: Resolver = async (\n`;
+      imports += `  transport: Transport,\n`;
+      imports += `  opts: GetAddressOptions\n`;
+      imports += `): Promise<Result> => {\n`;
+      imports += `  const signerFactory = (_: string) =>\n`;
+      imports += `    Promise.resolve(new polkadotSigner.default(transport));\n`;
+      imports += `  return ${family}Resolver(signerFactory)("", opts);\n`;
+      imports += `};\n`;
+      exprts += `\n  ${family},`;
+    }
```

Comment:
There seem to be some implied conventions on functions and / or file names. Could be wise to document them (if not done already).
Also, this whole file seems to get more and more complex. We might want / need to rethink it at some point (?)

Reply:
I agree this file still continue to grow and be more and more complex.
What we can do, it starts thinking about adding some more "utils" methods in the `coin-framework`.
WDYT?

Reply:
Why not, could be a good start. What kind of "utils" function? Like one (or more) to generate these "imports" / "exports" string?